### PR TITLE
Ignore weird backup filename on android

### PR DESCRIPTION
**Pull request by terrysahaidak**
Fixes RNBW-4500
Figma link (if any):

## Final checklist

- [ ] Assigned individual reviewers?
- [ ] Added labels? (`team1/team2`, `critical path`, `release`, `dev QA`)
- [ ] Did you test both iOS and Android?
- [ ] If your changes are visual, did you check both the light and dark themes?
- [ ] Added e2e tests? If not, please specify why
- [ ] If you added new critical path files, did you update the CODEOWNERS file?
- [ ] If no `dev QA` label, did you add the PR to the QA Queue?

### CODE REVIEW

**Review by perunt — APPROVED** [review]
Tested on an android phone, and it works well. The filename has the right format

**Review by tchayen — APPROVED** [review]
Code looks good

File: src/components/backup/RestoreCloudStep.js [hunk 1]
```diff
@@ -38,6 +39,7 @@ import Routes from '@/navigation/routesNames';
 import styled from '@/styled-thing';
 import { margin, padding } from '@/styles';
 import logger from '@/utils/logger';
+import { IS_ANDROID } from '@/env';
```
Comment (estrattonbailey) [inline src/components/backup/RestoreCloudStep.js:42]: 🙏

**Review by estrattonbailey — APPROVED** [review]

**Pull request closed without merging**